src/Explore.Cli/Program.cs

```diff
@@ -252,17 +261,61 @@ internal static async Task ExportSpaces(string exploreCookie, bool? verboseOutpu
                 return;
             }
 
+            var namesList = names?.Split(',')
+                .Select(name => name.Trim())
+                .ToList();
             var spaces = await spacesResponse.Content.ReadFromJsonAsync<PagedSpaces>();
             var panel = new Panel($"You have [green]{spaces!.Embedded!.Spaces!.Count} spaces[/] in explore");
             panel.Width = 100;
             panel.Header = new PanelHeader("SwaggerHub Explore Data").Centered();
+
+            // set the file name if provided
+            string fileName = "ExploreSpaces.json";
+            if (!string.IsNullOrEmpty(exportFileName))
+            {
+                // check if the file has a valid extension
+                if (!exportFileName.Contains('.'))
+                {
+                    fileName = $"{exportFileName}.json";
+                }
+                else if (exportFileName.EndsWith(".json"))
+                {
+                    fileName = exportFileName;
+                }
+                else
+                {
+                    AnsiConsole.MarkupLine($"[red]The file name provided has an invalid extension. Please review.[/]");
+                    return;
+                }
+
+                // check for invalid characters in the file name
+                if (exportFileName.Contains('\\') || exportFileName.Contains('/'))

```

Comment:
This is probably not an exhaustive check of invalid chars. It might be more robust to add some methods to UtilityHelper.cs to allow for robust checking.

Example:
```
    private static bool IsValidFileName(string fileName)
    {
        return (fileName.IndexOfAny(Path.GetInvalidFileNameChars()) >= 0);
    }

    private static bool IsValidFilePath(string filePath)
    {
        return (filePath.IndexOfAny(Path.GetInvalidPathChars()) >= 0);
    }
```

Reply:
Please add some UtilityHelperTests for above too 🙏